src/app.js

```diff
@@ -53,6 +55,8 @@ app.get('/', (req, res) => {
 });
 
 app.use('/api/images/', imagesRouter);
+app.use('/api/users/', usersRouter);
```

Comment:
Does order matter here?

Reply:
No I tested all the routes and had no issues. Especially since we wanna allow anonymous posting, even if images came before/after shouldn't actually impact anything